### PR TITLE
Dylan hilary nicole shara first PR

### PR DESCRIPTION
**Pull request by hilaryzen**
Created a markdown file with the team names

### CODE REVIEW

File: docs/first_pr/dylan_hilary_nicole_shara.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Our first PR
+
+Hilary Zen
+Shara Bhuiyan
+Nicole Khaimov
+
```
Comment (ryaanahmed) [inline docs/first_pr/dylan_hilary_nicole_shara.md:6]: Why is there an extra blank line here? Please fix.

**Review by ryaanahmed — CHANGES_REQUESTED** [review]
See comments inline -- just nitpicks.

**Comment by Dylan-Weber** [comment]
@ryaanahmed I removed the blank line as requested. Ready for review again.

**Review by ryaanahmed — APPROVED** [review]
@Dylan-Weber thanks for the quick fix! merging.

**Pull request merged**